jobs/log_parser/templates/bin/log_parser_ctl

```diff
@@ -26,11 +26,10 @@ case $1 in
     # construct a complete config file from all the fragments
     cat ${JOB_DIR}/config/input_redis_and_output_elasticsearch.conf > ${JOB_DIR}/config/logstash.conf
     cat ${JOB_DIR}/config/filters_pre.conf >> ${JOB_DIR}/config/logstash.conf
+    cat ${JOB_DIR}/config/logstash-filters-common.conf >> ${JOB_DIR}/config/logstash.conf
 
     <% if '' != p('logstash_parser.filters') %>
       cat ${JOB_DIR}/config/filters_override.conf >> ${JOB_DIR}/config/logstash.conf
-    <% else %>
-      cat ${JOB_DIR}/config/filters_default.conf >> ${JOB_DIR}/config/logstash.conf
```

Comment:
Note: this means we're no longer shipping some default filters that the experimental user can play with which I think we may be mentioned in the docs somewhere (which I'm perfectly okay with and support, but not something we previously discussed).